frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -384,7 +384,7 @@ pub enum ElectionCompute {
 	Signed,
```

Comment:
Please add a `#![deny(missing_docs)]` to the module, otherwise this will get oudated.